replications/service.go

```diff
@@ -76,6 +76,7 @@ type DurableQueueManager interface {
 	DeleteQueue(replicationID platform.ID) error
 	UpdateMaxQueueSize(replicationID platform.ID, maxQueueSizeBytes int64) error
 	CurrentQueueSizes(ids []platform.ID) (map[platform.ID]int64, error)
+	RemainingQueueSizes(ids []platform.ID) (map[platform.ID]int64, error)
```

Comment:
This change needs some updates to the tests as well so that they start building successfully again.